android/clientlib-ktx/build.gradle

```diff
@@ -29,11 +29,16 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    buildFeatures {
+        buildConfig = false
+    }
 }
 
 dependencies {
     api(project(":clientlib"))
     implementation 'androidx.core:core-ktx:1.8.0'
+    implementation "androidx.activity:activity-ktx:1.5.0"
```

Comment:
You might be able to delete this dependency now with the `ActivityResultSender`?